opal/mca/btl/tcp/btl_tcp_endpoint.c

```diff
@@ -62,6 +62,10 @@
 #include "btl_tcp_frag.h"
 #include "btl_tcp_addr.h"
 
+/*
+ * Magic ID string send during connect/accept handshake
+ */
+const char mca_btl_tcp_magic_id_string[] = "OPAL-TCP-BTL";
```

Comment:
These are 2 different issues. Here I was suggesting to add the BTL module/component version, to make sure we dont connect incompatible TCP modules.